src/Components/CardboardModal/CardboardModal.types.ts

```diff
@@ -0,0 +1,65 @@
+import {
+    IButtonProps,
+    IButtonStyles,
+    IIconStyles,
+    IModalStyles,
+    IStackStyles,
+    IStyle,
+    IStyleFunctionOrObject,
+    ITheme
+} from '@fluentui/react';
+import { CardboardIconNames } from '../../Models/Constants';
+
+export interface ICardboardModalProps {
+    /** on click of the destructive action. If not provided, button is hidden */
+    destructiveButtonProps?: IButtonProps;
```

Comment:
what is this "destructive" means in a modal?

Reply:
Hmm, so this is for actions to operations like "Delete" right? I just checked it on other websites and looks like "danger" is used more to refer that button is not necessarily destroying something but a dangerous action.

Reply:
Are you referring to the style for teh colors? Generally destroying things is considered dangerous when it comes to UX patterns because you can't undo it. 
Here is an example from ADO.
image.png

Reply:
![image](https://user-images.githubusercontent.com/57726991/193327463-c99bb81d-8753-40db-b8f4-43263b485186.png)

Reply:
oh! I didn't know this existed. Lovely!! Great find.

Reply:
Renamed.